shared/AppInsightsCore/src/JavaScriptSDK/CookieMgr.ts

```diff
@@ -190,6 +191,8 @@ export function createCookieMgr(rootConfig?: IConfiguration, logger?: IDiagnosti
                 let setCookieFn = cookieMgrConfig.setCookie || _setCookieValue;
                 setCookieFn(name, _formatCookieValue(theValue, values));
             }
+
+            return result;
```

Comment:
Do you forget to set result = true if cookie manager is enabled? Looks like the result will always be false.

Reply:
nice catch!

Reply:
Added